public/ball.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/ball.css">
+  <link rel="stylesheet" href="./css/button_outline_grow.css">
+  <script defer src="https://cdn.jsdelivr.net/npm/alpinejs@2.8.2/dist/alpine.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/Sortable/1.14.0/Sortable.min.js"></script>
+  <title>Hanabishi</title>
+</head>
+
+<body x-data="ball()" x-init="initSortable();">;
```

Comment:
[nits] なんかセミコロン入ってました 👀 
```suggestion
<body x-data="ball()" x-init="initSortable();">
```